app/models/parent_object.rb

```diff
@@ -490,10 +496,21 @@ def aspace_json=(a_record)
     self.barcode = a_record["orbisBarcode"]
   end
 
+  def sierra_json=(s_record)
+    super(s_record)
+    return s_record if s_record.blank?
+    self.bib = s_record["bibId"]
+    self.last_sierra_update = DateTime.current
+  end
+
   def ladybird_cloud_url
     "https://#{MetadataSource.metadata_cloud_host}/metadatacloud/api/#{MetadataSource.metadata_cloud_version}/ladybird/oid/#{oid}?include-children=1"
   end
 
+  def sierra_cloud_url
+    "https://metadata-api-test.library.yale.edu/metadatacloud/api/sierra/bib/#{bib}"
```

Comment:
We use `#{MetadataSource.metadata_cloud_host}` for the other source api's but it does not yet exist for sierra. This is hardcoded to `metadata-api-test` for now.

Reply:
This method should be a copy of `voyager_cloud_url` below, but change `ils` to `sierra` in the string that is returned.